tests/test_download_photos.py

```diff
@@ -322,7 +322,6 @@ def test_until_found(self):
         files_to_skip.append(("2018/07/30/IMG_7400.JPG", "photo", 2308885))
         files_to_skip.append(("2018/07/30/IMG_7400-medium.MOV", "photo", 1238639))
         files_to_skip.append(("2018/07/30/IMG_7399.JPG", "photo", 2251047))
-        files_to_download.append(("2018/07/30/IMG_7399-medium.MOV", "photo"))   # TODO this should not be downloaded, correct?
```

Comment:
Why did you remove this? Is that the file that causes the test issues for you?

Reply:
That file should never be downloaded, because all processing should stop after three skips. i added check to the test.